firmware/common2015/utils/logger/logger.cpp

```diff
@@ -5,18 +5,26 @@
 
 #include <mbed.h>
 #include <rtos.h>
+#include <cstdio>
+#include <errno.h>
 
 const char* LOG_LEVEL_STRING[] = {FOREACH_LEVEL(GENERATE_STRING)};
+const char* LOG_FILE_NAMES[] = {"/local/LOG1.TXT", "/local/LOG2.TXT"};
+
+const int maxLogSize = 500000;  // bytes
 
 /* The initial logging level shows startup info along with any
  * warning messages [but hopefully there's none of those :) ].
  */
 bool isLogging;  // = RJ_LOGGING_EN;
 
 uint8_t rjLogLevel;
+uint8_t logNameInd = 255;
 
 Mutex log_mutex;
 
+void findSmallestLogFile();
```

Comment:
Could you put this in logger.hpp? I /think/ that function definitions are supposed to go in hpp files to speed up build times.